.github/workflows/python-tests.yml

```
@@ -90,6 +90,27 @@ jobs:
         pip install .[testing,tensorflow]
 
     - run: pytest -Werror::FutureWarning -sv ./tests/test_keras*
+  
+  build_fastai:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.9"]
```

does fastai not support 3.10?
```suggestion
        python-version: ["3.7", "3.10"]
```

---

If we run this on a matrix of versions, that means 3 runners will run, eh? Is this even all that necessary? I see most the other ones seem to run just on 3.8.

---

I think running in different py versions is important here, specially as the pickling is different. Torch and TF integrations test 3.7 and 3.10, so I would do same here
https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-tests.yml#L55